src/Routes.js

```diff
@@ -73,7 +72,6 @@ const Routes = (props) => (
     <AuthenticatedRoute path="/admin_manage_institutions" component={AdminManageInstitutions} props={props} rolesAllowed={[USER_ROLES.admin]} />
     <AuthenticatedRoute path="/summary_votes" component={SummaryVotes} props={props} rolesAllowed={[USER_ROLES.all]} />
     <AuthenticatedRoute path="/researcher_console" component={ResearcherConsole} props={props} rolesAllowed={[USER_ROLES.researcher]} />
-    <AuthenticatedRoute path="/chair_console" component={ChairConsole} props={props} rolesAllowed={[USER_ROLES.chairperson]} />
     <AuthenticatedRoute path="/new_chair_console" component={NewChairConsole} props={props} rolesAllowed={[USER_ROLES.chairperson]} />
```

Comment:
Ideally, we should be redirecting folks to the `/chair_console` url, but the backing component would be `NewChairConsole`. Then, after `ChairConsole` is gone, we should rename `NewChairConsole` -> `ChairConsole` so that all traces of `new` are removed since it is no longer new.